src/main/java/org/folio/consortia/repository/UserTenantRepository.java

```diff
@@ -35,6 +35,10 @@ public interface UserTenantRepository extends JpaRepository<UserTenantEntity, UU
   @Query("DELETE FROM UserTenantEntity ut WHERE ut.userId= ?1 AND ut.isPrimary= true")
   void deleteByUserIdAndIsPrimaryTrue(UUID userId);
 
+  @Modifying
+  @Query("UPDATE UserTenantEntity ut SET ut.username= ?1 WHERE ut.userId= ?2")
+  void setUsernameByUserId(String username, UUID userId);
```

Comment:
We don't have logic to update username of shadow users now, we are updating only username of real user, the same we should do in mod-consortia, so we need to add tenantId param and user it in WHERE filtering as well.

Reply:
Added